docs/docs/recipes/protect-your-api/spring-boot.mdx

```diff
@@ -194,6 +195,8 @@ public class SecurityConfiguration {
     @Bean
     public JwtDecoder jwtDecoder() {
         NimbusJwtDecoder jwtDecoder = NimbusJwtDecoder.withJwkSetUri(jwksUri)
+                // Logto uses the ES384 algorithm to sign the JWTs by default.
```

Comment:
nit: Maybe also mention the signing-key rotation if they prefer the default RS256?

Reply:
Sure thing. Updated adding note section to link with the signing-key doc.